pydatarecognition/fsclient.py

```diff
@@ -238,8 +238,11 @@ def all_documents(self, collname, copy=True):
 
     def insert_one(self, dbname, collname, doc):
         """Inserts one document to a database/collection."""
-        coll = self.dbs[dbname][collname]
-        coll[doc["_id"]] = doc
+        try:
+            coll = self.dbs[dbname][collname]
+            coll[doc["_id"]] = doc
+        except TypeError:
```

Comment:
maybe don't catch the exception, let the program crash? Otherwise we have bad magic that the program is not updating the database but we don't know why.  Here having it crash with the exception is a feature, not a bug.